src/ripple/app/main/Main.cpp

```diff
@@ -403,6 +401,8 @@ int run (int argc, char** argv)
         "so this option is not needed for users")
     ("unittest-child",
         "For internal use only when spawning child unit test processes.")
+    ("fg",
+        "Depreceated: server always in foreground mode.")
```

Comment:
"Deprecated"
* https://www.dictionary.com/browse/deprecate
* https://en.wikipedia.org/wiki/Deprecation

Reply:
D’oh! I know better than that.